README.md

```diff
@@ -371,6 +371,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
 | Lucy                          | [cometkim/asdf-lucy](https://github.com/cometkim/asdf-lucy)                                                     | [![CI](https://github.com/cometkim/asdf-lucy/actions/workflows/ci.yml/badge.svg)](https://github.com/cometkim/asdf-lucy/actions/workflows/ci.yml)                                                                                 |
 | mage                          | [mathew-fleisch/asdf-mage](https://github.com/mathew-fleisch/asdf-mage)                                         | [![Build Status](https://github.com/mathew-fleisch/asdf-mage/workflows/Build/badge.svg?branch=main)](https://github.com/mathew-fleisch/asdf-mage/actions)                                                                         |
 | make                          | [yacchi/asdf-make](https://github.com/yacchi/asdf-make)                                                         | [![Build Status](https://github.com/yacchi/asdf-make/actions/workflows/build.yml/badge.svg)](https://github.com/yacchi/asdf-make/actions?query=workflow%3ABuild)                                                                  |
+| mani                          | [anweber/asdf-mani](https://github.com/anweber/mani)                                                         | [![Build Status](https://github.com/anweber/asdf-mani/actions/workflows/build.yml/badge.svg)](https://github.com/anweber/asdf-mani/actions?query=workflow%3ABuild)                                                                  |
```

Comment:
This first link here should point to the plugin and not the tool of the plugin

Reply:
sorry for the typo. Url is fixed. thx